main.go

```diff
@@ -38,6 +38,8 @@ var (
 	enableMermaid     = kingpin.Flag("mermaid", "use mermaid diagrams where possible").Default("false").Bool()
 	enableRedoc       = kingpin.Flag("redoc", "generate redoc for specs imported from openapi. Must be run on a git repo.").Default("false").Bool()
 	imageDest         = kingpin.Flag("imageDest", "Optional image directory destination (can be outside output)").String()
+	feedbackLink      = kingpin.Flag("feedback", "").Default("https://github.com/anz-bank/sysl-catalog/issues/new").String()
+	chatLink          = kingpin.Flag("chat", "").String()
```

Comment:
Should we set the default for chatLink to a channel in the ANZOSS Slack?

Reply:
Good idea